src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle'
+  color: Color
+  getArea: ()=>{}
+}
+enum FigureForm {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+function getTriangle(a:number, b:number, c:number) : boolean {
```

Comment:
The function name 'getTriangle' is not clear. It should be something like 'isTriangle' because it checks whether three given sides can form a triangle and returns a boolean.